bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -311,8 +311,11 @@ spec:
                   return
             fi
             echo "Checking ${db} health"
-            serverid=$(ovsdb-tool show-log "${db}" | sed -ne '/server_id:/s/server_id: *\([[:xdigit:]]\+\)/\1/p')
-            match=$(ovsdb-tool show-log "${db}" | grep "prev_servers: *${serverid}(" || true)
+            # remove leading and trailing space around server_id
+            serverid=$(ovsdb-tool show-log "${db}" | sed -ne '/server_id:/s/ *server_id: *\([[:xdigit:]]\+\).*/\1/p')
+            # prev_servers is a list we are searching for " $server_id(" in the list
```

Comment:
Either that needs to be two sentences (period before "we") or the "the list" at the end is superfluous.